server/src/main/java/org/elasticsearch/action/support/master/TransportMasterNodeAction.java

```diff
@@ -76,7 +76,7 @@ protected TransportMasterNodeAction(
         Writeable.Reader<Request> request,
         IndexNameExpressionResolver indexNameExpressionResolver,
         Writeable.Reader<Response> response,
-        String executor
+        Executor executor
```

Comment:
Serverless has a few callers of this so we need to keep the old API around until we can fix them too.